apps/election-manager/test/helpers/fakeKiosk.ts

```diff
@@ -43,22 +43,18 @@ export function fakePrinterInfo(
  */
 export default function fakeKiosk({
```

Comment:
shouldn't this just be deleted if we're using the test-utils fakeKiosk now?

Reply:
Yep, good catch.